metadata/metadata.yml

```diff
@@ -1,5 +1,5 @@
 metadata_version: 1
-name: aind-multiplane-ophys-pipeline without splitting v6 extraction
+name: TEST multi-inputs aind-multiplane-ophys-pipeline without splitting v6 extraction
```

Comment:
Ain't a test anymore but serious business :), thus don't change the pipeline name, all good otherwise!